twatbot.py

```diff
@@ -643,11 +659,14 @@ def httpsrv_client_thread(c, evt_done):
 		c.send(200, "OK", "User-agent: *\nDisallow: /")
 
 	elif req['url'].startswith('/config.html'):
-		vars=vars_from_request(req)
-		r, redir = configpage(req,vars)
+		if args.config > 0:
+			vars=vars_from_request(req)
```

Comment:
we may want to change that at some point, `vars` is a reserved identifier in python...